src/main/java/racingcar/InputView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String ASK_CAR_MENT = "자동차 대수는 몇 대 인가요?";
+    private static final String ASK_ROUND_MENT = "시도할 회수는 몇 회 인가요?";
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
`static final` 키워드가 있는 상수이니, 컨벤션에 따라 이름을 대문자로 작성하면 완벽할 것 같습니다.
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```